scripts/gulpfiles/build_tasks.js

```diff
@@ -449,6 +449,7 @@ function buildAdvancedCompilationTest() {
     'tests/compile/main.js',
     'core/**/**/*.js',
     'blocks/*.js',
+    'tests/compile/test_blocks.js',
```

Comment:
Maybe move up so this is right after main.js (for association)

Reply:
done